src/Domain/CommentRepository.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+interface CommentRepository {
+
+	/**
+	 * @param int $limit
+	 *
+	 * @return Comment[]
+	 */
+	public function getComments( $limit );
```

Comment:
Should't there be an optional param for the start offset so we can get comment paging?